conductor/server/workflows/decommission_device.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "decommission_device",
+    "description": "Removes device info from all services.",
+    "version": 1,
+    "tasks": [
+        {
+            "name": "delete_device_inventory",
+            "taskReferenceName": "delete_device_inventory",
+            "inputParameters": {
+                "http_request": {
+                    "uri": "http://mender-inventory:8080/api/0.1.0/devices/${workflow.input.device_id}",
+                    "method": "DELETE"
```

Comment:
done, the workflow accepts a `request_id` param and passes it on via the usual `X-MEN-RequestID` header.

also @kjaskiewiczz